src/js/tabs/onboard_logging.js

```diff
@@ -350,6 +350,10 @@ onboard_logging.initialize = function (callback) {
                 {text: "RX_EXPRESSLRS_SPI"},
                 {text: "RX_EXPRESSLRS_PHASELOCK"},
                 {text: "RX_STATE_TIME"},
+                {text: "DEBUG_GPS_RESCUE_VELOCITY"},
+                {text: "DEBUG_GPS_RESCUE_HEADING"},
+                {text: "DEBUG_GPS_RESCUE_TRACKING"},
+                {text: "DEBUG_ATTITUDE"},
```

Comment:
Here we don't use the DEBUG word. It's a list of debug modes, so please remove it.
```suggestion
                {text: "GPS_RESCUE_VELOCITY"},
                {text: "GPS_RESCUE_HEADING"},
                {text: "GPS_RESCUE_TRACKING"},
                {text: "ATTITUDE"},
```

Reply:
Thanks. Updated.